lib/fullstory/index.js

```diff
@@ -67,6 +68,11 @@ FullStory.prototype.identify = function(identify){
     del(traits, 'name');
   }
 
+  if (traits.Email) {
```

Comment:
Yeah it's not. Here's an example: http://cl.ly/image/1Q3q3d0P2d3p/Screen%20Shot%202015-02-02%20at%203.51.52%20PM.png